pkg/imports/imports.go

```diff
@@ -14,11 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// The signals package contains libraries for handling signals to shutdown the system.
-package signals
+/*
+Package imports ensures that when "dep ensure" is run after creating a new kubebuilder project, it
+will fetch all of the dependencies needed for the project.
+*/
+package imports
```

Comment:
hmm....this could create problem with one scenario: If user runs `vendor update` on an existing project, his project will no longer depend the `dummy imports` because their project don't have have pkg `hack` ?

Reply:
I think that is fine.  If a user has an existing project then they probably won't run into the boot strap problem this is meant to solve.

Reply:
thinking more.... if user run `dep ensure` on their project after updating to new kubebuilder, direct dependency for `generate docs` might be gone and then `generate docs` will fail ?